example/src/Client/utils/BlockProcessor.ts

```diff
@@ -123,15 +109,17 @@ export class BlockProcessor {
   }
 
   private async _processBlock(block: LightBlock) {
+    this.blockCache.cacheBlock(block);
+
     const notes: NoteEncrypted[] = [];
```

Comment:
Can we change the function signature of the `addNotesToMerkleTree` to just take in the type of output.note[]? I just want to encapsulate all the sdk calls to the merkle tree so that it will be clear how to handle clients without the sdk.

I know you didn't make this code, so feel free to say no or push out :)